bin/stage

```diff
@@ -33,15 +32,36 @@
 # Configuration
 # ========================================
 
+# Load in configuration file, or ask and populate.
+def read_or_ask_for_configuration
+  require 'yaml'
+  conf_file = File.join(ENV['HOME'], '.jekyll-stagingrc')
```

Comment:
Now done in https://github.com/mattgemmell/Jekyll-Staging/commit/4ea7e4cee72b20d48d651cb35410a52dfcd00d5e